### PR TITLE
Swift sample update

### PR DESCRIPTION
**Pull request by unpluggedk**

### CODE REVIEW

File: MSAL/src/public/MSALAuthority.h [hunk 1]
```diff
@@ -31,12 +31,13 @@
 
 @interface MSALAuthority : NSObject
 
-@property (readonly, nonnull) NSURL *url;
+NS_ASSUME_NONNULL_BEGIN
+@property (readonly) NSURL *url;
 
-- (instancetype _Nullable )init NS_UNAVAILABLE;
-+ (instancetype _Nullable )new NS_UNAVAILABLE;
+- (instancetype _Null_unspecified )init NS_UNAVAILABLE;
```
Comment (oldalton) [inline MSAL/src/public/MSALAuthority.h]: why _Null_unspecified?

File: Samples/ios/SampleAppiOS-Swift/SampleAppiOS-Swift.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -172,21 +186,23 @@
 			isa = PBXGroup;
 			children = (
 				96672DB71EC7913D00878AC5 /* MSAL.framework */,
-				96672DB91EC7913D00878AC5 /* "MSAL-iOS-Unit-Tests".xctest */,
+				96672DB91EC7913D00878AC5 /* "MSAL-iOS-UI-Tests".xctest */,
```
Comment (oldalton) [inline Samples/ios/SampleAppiOS-Swift/SampleAppiOS-Swift.xcodeproj/project.pbxproj]: why did we rename Unit to UI?

File: Samples/ios/SampleAppiOS-Swift/SampleAppiOS-Swift/SamplePhotoUtil.swift [hunk 3]
```diff
@@ -235,3 +235,9 @@ fileprivate extension SamplePhotoUtil {
     
     
 }
+
+// Helper function inserted by Swift 4.2 migrator.
```
Comment (oldalton) [inline Samples/ios/SampleAppiOS-Swift/SampleAppiOS-Swift/SamplePhotoUtil.swift]: what is that for :)?

File: Samples/ios/SampleAppiOS-Swift/SampleAppiOS-Swift.xcodeproj/project.pbxproj (hunk 2 above)
Comment (unpluggedk) [reply Samples/ios/SampleAppiOS-Swift/SampleAppiOS-Swift.xcodeproj/project.pbxproj]: Looks weird, I will set it back

File: Samples/ios/SampleAppiOS-Swift/SampleAppiOS-Swift/SamplePhotoUtil.swift (hunk 3 above)
Comment (unpluggedk) [reply Samples/ios/SampleAppiOS-Swift/SampleAppiOS-Swift/SamplePhotoUtil.swift]: Inserted by default.

File: MSAL/src/public/MSALAuthority.h (hunk 1 above)
Comment (unpluggedk) [reply MSAL/src/public/MSALAuthority.h]: The init() method defined in the NSObject class does no initialization; it simply returns self. In terms of nullability, callers can assume that the NSObject implementation of init() does not return nil.

**Pull request merged**